docs/components/box/index.html

```diff
@@ -43,9 +41,12 @@ <h2 class="hxSectionTitle" id="demo">Demo</h2>
             </p>
           </div>
         </div>
-        <span class="hxCaption">
+        <small>
           *Visual styling added for demonstration purpopses.
-        </span>
+        </small>
+        {% code 'html' %}{% raw %}
```

Comment:
Readability note: I'd put `{% raw %}` down on the next line at the start; otherwise a quick glance through makes it look like the start tag is missing.

Reply:
This is intentional, because otherwise the syntax highlighting will put the first line of code at a different indentation level as everything else.